docs/standard/io/media/pipelines/code/DoNotUse.cs

```diff
@@ -14,8 +13,12 @@ private void Dummy()
             Environment.FailFast("This code is terrible, don't use it!");
             while (true)
             {
-                ReadResult result = await reader.ReadAsync(cancellationToken);
-                ReadOnlySequence<byte> dataLossBuffer = result.Buffer;
+                var pipe = new Pipe();
```

Comment:
Revert all of these code changes. Take the PipeReader as method input but don’t new it up in the while loop.